src/labels/CustomLabels.labels

```diff
@@ -89,6 +89,14 @@
         <value>Male</value>
     </labels>
     <labels>
+        <fullName>PreferredEmailMatchMustExist</fullName>
+        <categories>Contacts</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>PreferredEmailMatchMustExist</shortDescription>
+        <value>The value selected in Preferred Email does not match any email fields. Values in Preferred Email must match a custom email field label.</value>
```

Comment:
I see, I think. I was confused about where this error gets displayed. Would this work? "We can't find the email specified in Preferred Email. Make sure that you're entering the field label of an existing custom email field." @davidjray